apps/ae_socket_connector/test/ae_socket_connector_test.exs

```diff
@@ -1236,6 +1236,80 @@ defmodule SocketConnectorTest do
     )
   end
 
+  @tag :force_progress
+  test "force progress job", context do
+    {alice, bob} = gen_names(context.test)
+
+    scenario = fn {initiator, intiator_account}, {responder, responder_account}, runner_pid ->
+      initiator_contract =
+        {TestAccounts.initiatorPubkeyEncoded(), "../../contracts/TicTacToe_old.aes",
+         %{abi_version: 1, vm_version: 3, backend: :aevm}}
+
+      [
+        {:initiator,
+         %{
+           message: {:channels_update, 1, :self, "channels.update"},
+           next: {:async, fn pid -> SocketConnector.initiate_transfer(pid, 2) end, :empty},
+           fuzzy: 10
+         }},
+        {:initiator,
+         %{
+           message: {:channels_update, 2, :self, "channels.update"},
+           fuzzy: 3,
+           next:
+             ClientRunnerHelper.assert_funds_job(
+               {intiator_account, 6_999_999_999_997},
+               {responder_account, 4_000_000_000_003}
+             )
+         }},
+        {:initiator,
+         %{
+           next: {:async, fn pid -> SocketConnector.new_contract(pid, initiator_contract, 10) end, :empty}
+         }},
+        {:initiator,
+         %{
+           fuzzy: 10,
+           message: {:channels_update, 3, :self, "channels.update"},
+           next:
+             {:async,
+              fn pid ->
+                SocketConnector.force_progress(
+                  pid,
+                  initiator_contract,
+                  'make_move',
+                  ['11', '1']
+                )
+              end, :empty}
+         }},
+        {:initiator,
+         %{
+           # "consumed_forced_progress", comming as on_chain means, that FP was executed successfully
+           message: {:on_chain, "consumed_forced_progress"},
```

Comment:
would also expect, or is there a documentation missmatch?
```
{
  "jsonrpc": "2.0",
  "method": "channels.on_chain_tx",
  "params": {
    "channel_id": "ch_2eiGsAvt...",
    "data": {
      "info": "channel_changed",
      "tx": "tx_+QFxCwH4QrhA...",
      "type": "channel_force_progress_tx"
    }
  },
  "version": 1
}
```

Reply:
Yes, it should also expect `channels.on_chain_tx`, my fault, as I decided to suppress it...

Reply:
Could we please add that to the test case for the sake of completeness